.github/actions/setup-externals/action.yaml

```diff
@@ -112,10 +110,10 @@ runs:
         fi
         make -j4
         make install
-      elif [[ "${{ inputs.mpi }}" == "mpich"* ]]; then
-        wget http://www.mpich.org/static/downloads/${MPICH_VERSION}/mpich-${MPICH_VERSION}.tar.gz
-        tar -xzf mpich-${MPICH_VERSION}.tar.gz
-        cd mpich-${MPICH_VERSION}
+      elif [[ "${MPI_NAME}" == "mpich" ]]; then
+        wget http://www.mpich.org/static/downloads/${MPI_VERSION}/mpich-${MPI_VERSION}.tar.gz
+        tar -xzf mpich-${MPI_VERSION}.tar.gz
+        cd mpich-${MPI_VERSION}
```

Comment:
Not that it's a problem, but is there a reason not to install openmpi/mvapich2 through spack?

Reply:
The driver for that was that building MPI takes about 45mins and we are already hitting the 6hr walltime. Building it outside of spack-stack allows us to run it as a separate step (setup externals) and cache it. Reduces the times for iterating over build failures for spack packages by a lot.